dandi/organize.py

```diff
@@ -254,6 +254,8 @@ def organize_external_files(
                     ext_file_dict["external_files_renamed"],
                 )
             ):
+                if any([str(name_old).startswith(i) for i in ["http", "ftp"]]):
```

Comment:
can't have `ftpdownloaded.mp3`? check at least (and no need for `[]` AFAIK) for 

```suggestion
                if any(str(name_old).startswith(i) for i in ["http://", "https://", "ftp://"]):
```